src/h5web/toolbar/controls/DomainSlider/DomainSlider.tsx

```diff
@@ -65,13 +69,12 @@ function DomainSlider(props: Props): ReactElement {
         thumbClassName={styles.thumb}
         thumbActiveClassName={styles.thumbActive}
         pearling
-        value={appliedDomain}
-        min={extendedMin}
-        max={extendedMax}
-        step={step}
+        min={0}
+        max={100}
```

Comment:
Ideally, `min` and `max` should be set from `sliderScale.range`

Reply:
Ha, good point, especially since `min` should be `1` here...